### PR TITLE
(WIP) Update tika to 2.3.0

### PR DESCRIPTION
**Pull request by smcgrattan**
Code changes finished. POMs will need to be updated when ready to merge and then rebased.

### CODE REVIEW

**Comment by cfkoehler** [comment]
> Code changes finished. POMs will need to be updated when ready to merge and then rebased.

I might be missing some context. Why have the PR with the '7.7.0-tika-update-SNAPSHOT' version?

**Comment by cfkoehler** [comment]
@smcgrattan Have you verified that these dependencies are available in our production environment? Should not be a problem to get them if not but good heads up if we merge and then get a build failure.

**Review by cfkoehler — APPROVED** [review]

**Comment by smcgrattan** [comment]
only selected versions available in production

**Comment by smcgrattan** [comment]
that's a temporary version id for the WIP so it can be specified in upstream poms to ensure proper selection.

**Comment by cfkoehler** [comment]
@smcgrattan Is this still draft? 
I lost track of where we were going with Tika. 
Should I do a full review on this?

**Comment by cfkoehler** [comment]
@smcgrattan 
Is there anything blocking you from getting this to completion? 
Updating tika will solve most of our security vulnerabilities by transitive dependencies it brings in.

File: pom.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>emissary</groupId>
   <artifactId>emissary</artifactId>
-  <version>7.7.0-SNAPSHOT</version>
+  <version>7.7.0-tika-update-SNAPSHOT</version>
```
Comment (cfkoehler) [inline pom.xml:6]: Remove this change of version

**Review by cfkoehler — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (smcgrattan) [reply pom.xml:6]: i'm working on updating it now. What's there now is very much out of date

**Comment by pechrista** [comment]
lgtm, will approve once pom files are updated

**Pull request closed without merging**